Example/Emission/AuthenticationManager.m

```diff
@@ -119,5 +130,20 @@ - (void)authenticateWithEmail:(NSString *)email password:(NSString *)password ke
   }];
 }
 
+- (void)validateStoredCredentialsPassing:(dispatch_block_t)pass failing:(dispatch_block_t)fail
+{
+  ArtsyNetworkOperator *networkOperator = [ArtsyNetworkOperator new];
+  ArtsyAuthentication *auth = [[ArtsyAuthentication alloc] initWithClientID:@"e750db60ac506978fc70"
+                                                               clientSecret:@"3a33d2085cbd1176153f99781bbce7c6"];
+  auth.router.authToken = [[ArtsyToken alloc] initWithToken:self.token expirationDate:[NSDate distantFuture]];
+
+  NSURLRequest *request = [auth.router requestForUserDetails];
+  [networkOperator JSONTaskWithRequest:request success:^(NSURLRequest *_, NSHTTPURLResponse *__, id JSON) {
+    pass();
+  } failure:^(NSURLRequest *_, NSHTTPURLResponse *response, NSError *error, id JSON) {
+    fail();
+  }];
+}
```

Comment:
pulls out some of the authentication objects to make it use the current setup for validation